script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -315,18 +315,18 @@ Sensei ran up to me and slapped me on the cheek.
 "Stop it. You've just done something incredibly reckless."
 <Page89>
 Sensei looked at me with serious eyes.
-Relentless and unwavering eyes that punish wrongdoers.
+Relentless and unwavering eyes that were punishing wrongdoers.
```

Comment:
I think that `punish` is still correct here, the plain form of the verb is being used as an adjective, so I don't think you want to change it.

Reply:
I second ross' comment here